### ISSUE TITLE
NaN in histogram 

### ISSUE THREAD

**Issue by knutole** [systemapic/systemapic.js#111]
If dragging all the way to the left, `NaN` appears in SQL query.

![screen shot 2016-02-17 at 5 50 33 pm](https://cloud.githubusercontent.com/assets/2197944/13117160/1d3a6802-d59f-11e5-97f7-1661cf1bf4da.png)

**Comment by jorgenevil** [comment]
I cannot reproduce this error. Is it fixed? If not, can I see a dataset where this is happening?

**Comment by knutole** [comment]
You need to drag the slider all the way to the left.

[points_geom_z.zip](https://github.com/systemapic/systemapic.js/files/141283/points_geom_z.zip)

**Comment by knutole** [comment]
This happens on columns which already are a bit suspicious. Note for example the x-axis has value 7 on all steps. 

Nothing wrong with histogram-code per se, but should detect bad columns better.

**Comment by jorgenevil** [comment]
Yes, I've been looking at some datasets, and it seems to work fine with data that has numeric columns most of the time. 

I haven't really looked into this, but can we get information in the metadata of what sort of data that each column contains? I know that shapefiles are quite specific about this, telling you if it's a `String`, `Real`, `Integer`, `Date` etc, plus allowed length. If it's easy to pull this information and pass it into the meta for the layer, it might help this issue at the same time as it would be very helpful in the styler.

Screenshot of property table of file above in QGis
![screen shot 2016-02-23 at 8 16 23 pm](https://cloud.githubusercontent.com/assets/8961926/13263590/6d2e40ac-da6a-11e5-9cad-3098410cf706.png)
